test/message/timeout_throw.chakracore.out

```diff
@@ -1,4 +1,5 @@
 ReferenceError: 'undefined_reference_error_maker' is undefined
    at Anonymous function (*test*message*timeout_throw.js:*:*)
+   at ontimeout (timers.js:*:*)
```

Comment:
It's just a refactoring in lib/timers.js.  A new timeout method was introduced before calling the user callback.